File: Vault/Content/Js/types/CredentialSummary.ts
export class CredentialSummary {
    constructor(
        public credentialid: string,
        public masterkey: string,
        public userid: string,
        public description: string,
        public username: string,
        public password: string,
        public url: string,
        public weak: boolean) {
    }
}

File: Vault/Models/CredentialSummary.cs
using Newtonsoft.Json;

namespace Vault.Models
{
    public class CredentialSummary
    {
        [JsonProperty("credentialID")]
        public string CredentialID { get; set; }

        [JsonProperty("description")]
        public string Description { get; set; }

        [JsonProperty("username")]
        public string Username { get; set; }

        [JsonProperty("password")]
        public string Password { get; set; }

        [JsonProperty("url")]
        public string Url { get; set; }
    }
}
